src/tox/config/__init__.py

```diff
@@ -410,6 +408,19 @@ def __setitem__(self, name, value):
         self.definitions[name] = value
         self.resolved[name] = value
 
+    def items(self):
+        return [(name, self.__getitem__(name)) for name in self.definitions]
```

Comment:
```suggestion
        return ((name, self[name]) for name in self.definitions)
```

1. more closely match the python3 `.items()` api
2. don't call double-double-under methods